src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -63,9 +63,14 @@ export class Conversations extends React.Component<Props, State> {
       return
     }
 
-    this.setState({ fetchingNextPage: true })
+    this.setState({
+      fetchingNextPage: true,
+    })
+
     this.props.relay.refetchConnection(10, () => {
-      this.setState({ fetchingNextPage: false })
+      this.setState({
+        fetchingNextPage: false,
+      })
```

Comment:
I wonder if `this.prop.relay.isLoading()` properly reflects that it is refetching, in which case all of this `fetchingNextPage` state faffing can go.

Reply:
Worked like a charm

Reply:
Dope 👍 I’ve just been replacing it whenever I touch code that has the old pattern.